modules/ROOT/pages/query/procedures.adoc

```diff
@@ -0,0 +1,164 @@
+= Aura Built-in Procedures Reference
+
+This page provides a reference for built-in procedures supported in Neo4j Aura (including both AuraDB and AuraDS). 
+Aura restricts access to certain administrative and custom capabilities due to its fully managed, secure architecture. 
+This guide highlights what is available, how to use them, and which procedures are not supported.
+
+The full list of Neo4j procedures is available at https://neo4j.com/docs/operations-manual/current/procedures/
+
+To check which procedures are available in your Aura instance, run the following Cypher command in the query editor or Cypher shell:
+
+.List available procedures with default output columns
+[source,cypher]
+----
+SHOW PROCEDURES;
+----
+
+.All procedures must be called with the CALL keyword, e.g.
+[source,cypher]
+----
+CALL db.labels();
+----
+
+== Supported Built-in Procedures
+
+The following procedures are available in Aura by default:
+
+== Schema and Metadata
+
+`CALL db.labels();`
```

Comment:
```suggestion
`db.labels();`
```
We are listing the procedures, seems superfluous to include `CALL` on every line.